src/core/qgsproject.cpp

```diff
@@ -2133,4 +2133,13 @@ QMap<QString, QgsMapLayer *> QgsProject::mapLayers() const
   return mLayerStore->mapLayers();
 }
 
+QgsTransactionGroup *QgsProject::transactionGroup(const QString &providerKey, const QString &connString)
+{
+    QgsTransactionGroup *transaction = mTransactionGroups.value( qMakePair( providerKey, connString ) );
+
+    if (!transaction)
```

Comment:
I don't think it's worth adding this message. It should be straightforward to guess what happened if a nullptr is returned. Instead it would be good to document this in the API docs.